### DISCUSSION TITLE
Security Onion 2.4 active response

### DISCUSSION THREAD

**Discussion by bmbeverst** [Security-Onion-Solutions/securityonion#10734] [2.4]
Good day, I have not seen this [question asked](https://github.com/Security-Onion-Solutions/securityonion/discussions?discussions_q=Security+Onion+2.4+active+response) or any clear answer.

Since Wazuh is being dropped, is Security Onion 2.4 going to have an active response? I admit I am biased towards Wazuh. It seems [Elastic Agent](https://www.elastic.co/guide/en/security/current/response-actions.html) does have the feature, but you need to have an Enterprise subscription to use it. 

Thanks

**Comment by InfosecGoon** [comment]
Could you expand on your use case a little bit? Are you just trying to blocklist potentially malicious SSH connections or do you have something else in mind?

**Reply by bmbeverst** [reply]
I parse Suricata logs with Wazuh and then add blocks to iptables when a high alert occurs, over 13. I tested with port scanning, and the host that was scanning my Wazuh agent was then blocked.

Furthermore, I am just learning how to use the active response, and there are plenty of example on the internet of other users using it for interesting solutions. It can also be used to [block](https://documentation.wazuh.com/current/user-manual/capabilities/active-response/ar-use-cases/blocking-ssh-brute-force.html) ssh brute forcing. [Disable](https://documentation.wazuh.com/current/user-manual/capabilities/active-response/ar-use-cases/disabling-user-account.html) a Linux user after failed login attempts. [Delete](https://wazuh.com/blog/detecting-and-responding-to-malicious-files-using-cdb-lists-and-active-response/) malware on a host.
